unified-runtime/source/adapters/hip/image.cpp

```diff
@@ -1003,10 +1001,6 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageGetInfoExp(
     return UR_RESULT_ERROR_INVALID_VALUE;
   }
 #else
-  std::ignore = hImageMem;
-  std::ignore = propName;
-  std::ignore = pPropValue;
-  std::ignore = pPropSizeRet;
```

Comment:
Some more ifdefd unused

Reply:
This one's a `HIP_VERSION >= 50600000` so it still needs some `[[maybe_unused]]`

Reply:
thanks

Reply:
fixed